src/video_core/command_classes/codecs/h264.h

```diff
@@ -48,7 +50,7 @@ class H264BitWriter {
 
     /// Based on section 7.3.2.1.1.1 and Table 7-4 in the H.264 specification
     /// Writes the scaling matrices of the sream
-    void WriteScalingList(const std::vector<u8>& list, s32 start, s32 count);
+    void WriteScalingList(const std::span<u8>& list, s32 start, s32 count);
```

Comment:
A span is a view of the underlying bytes
```suggestion
    void WriteScalingList(std::span<const u8> list, s32 start, s32 count);
```